components/support/migration/src/main/java/mozilla/components/support/migration/FennecMigrator.kt

```diff
@@ -70,6 +71,11 @@ sealed class Migration(val currentVersion: Int) {
      * Migrates all Fennec settings backed by SharedPreferences.
      */
     object Settings : Migration(currentVersion = 1)
+
+    /**
+     * Migrates / Disables all currently unsupported Add-ons.
+     */
+    object Addons : Migration(currentVersion = 1)
```

Comment:
You'll also need to add this migration to `MigrationResultsStore#fromJSON`, otherwise we'll crash when reading the persisted migrations.

Reply:
This has come up every time we add a migration. We should have modeled this in a way that the type system enforces it. But, well, I also do not want to refactor anything that late anymore.

Reply:
Ah, thanks, good catch! Updated.

Reply:
Totally agreed @pocmo, both that this is bad now, and that it's not really worth refactoring at this point.